src/components/App.js

```diff
@@ -4,20 +4,60 @@ import PropTypes from "prop-types";
 import "./App.css";
 import ListOfUsers from "./ListOfUsers";
 
+
 class App extends Component {
 
   constructor(props) {
     super(props);
 
     this.state = {
       addText: "Select User",
-      removeText: null,
+      removeText: "",
       message: "No users selected",
       selectedUserList: [],
       userList: this.props.users
     };
   }
 
+  selectUser(selectedUser) {
+    this.setState({
+      message: null,
+      removeText: "Remove User",
+      userList: [
+        ...this.state.userList].filter((item) => {
+          return item !== selectedUser;
+        }),
+      selectedUserList: [
+        ...this.state.selectedUserList,
+        selectedUser
+      ],
+    });
+  }
+
+  removeUser(selectedUser) {
+    this.setState({
+      selectedUserList: [
+        ...this.state.selectedUserList].filter((item) => {
```

Comment:
Check the syntax here. You are creating a copy of selectedUserList. This is unnecessary since filter returns a new array anyways.